deployment-platforms/render/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "prisma-on-render",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Example Express API that uses Prisma ORM, deployed on Render",
+  "main": "index.js",
+  "scripts": {
+    "build": "prisma generate",
+    "dev": "PORT=3000 node src/index.js",
+    "start": "node src/index.js"
+  },
+  "author": "@jess-render",
+  "license": "ISC",
+  "dependencies": {
+    "@prisma/client": "5.21.1",
+    "body-parser": "1.20.3",
+    "express": "4.21.1",
+    "prisma": "5.21.1"
```

Comment:
prisma is usually a dev dependency

Reply:
I did this for a very specific reason... but with a bit of digging, I found a better solution. Will update!

Long version:
Today Render doesn't have a distinction between "build dependencies" and "prod dependencies".

If someone sets NODE_ENV to `production` in their service, npm install would skip the devDependencies, which are needed for the build step.

But TIL about `--production=false`

> With the --production flag (or when the NODE_ENV environment variable is set to production), npm will not install modules listed in devDependencies. To install all modules listed in both dependencies and devDependencies when NODE_ENV environment variable is set to production, you can use --production=false.
from https://docs.npmjs.com/cli/v10/commands/npm-install